src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1756,4 +1756,10 @@ This version used in: {2}</value>
   <data name="Implement_explicitly" xml:space="preserve">
     <value>Implement explicitly</value>
   </data>
+  <data name="A_source_file_is_missing_a_required_header" xml:space="preserve">
+    <value>A source file is missing a required header.</value>
+  </data>
+  <data name="The_file_header_is_missing_not_located_at_the_top_of_the_file_or_does_not_match_the_required_text" xml:space="preserve">
+    <value>The file header is missing, not located at the top of the file, or does not match the required text</value>
```

Comment:
coudl we break these up?

Reply:
It would be easy to break it up into two cases:

1. The file header is missing or not located at the top of the file
2. The file header does not match the required text

Reply:
➡️ Now covered by other comments